testsuite/tests/lib-unix/win-env/stubs.c

```diff
@@ -18,3 +18,11 @@ CAMLprim value caml_SetEnvironmentVariable(value s1, value s2)
   caml_stat_free(w2);
   return Val_unit;
 }
+
+CAMLprim value stub_putenv(value s)
```

Comment:
Is there a reason to define this function, now that Unix.putenv relies on _wputenv again?

Reply:
No, there isn't, good catch!